lib/Doctrine/DBAL/Cache/ArrayStatement.php

```diff
@@ -138,11 +138,8 @@ public function fetchAll($fetchMode = null, $fetchArgument = null, $ctorArgs = n
     public function fetchColumn($columnIndex = 0)
     {
         $row = $this->fetch(PDO::FETCH_NUM);
-        if (!isset($row[$columnIndex])) {
-            // TODO: verify this is correct behavior
-            return false;
-        }
 
-        return $row[$columnIndex];
+        // TODO: verify this is correct behavior
```

Comment:
After having two lines smashed together, it's not clear what exactly should be verified. Previously it was clear it was about returning `false`. You may consider updating the comment.